### PR TITLE
IOS magnifying glass

### PR DESCRIPTION
**Pull request by caesar-exploit**
IOS is over-emphasizing the magnifying glass

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: sites/kit.svelte.dev/src/lib/search/SearchBox.svelte [hunk 1]
```diff
@@ -231,6 +231,7 @@
 		border: none;
 		border-bottom: 1px solid #eee;
 		font-weight: 600;
+		appearance: none;
```
[the remark this answers is not in the document]
Comment (Rich-Harris) [reply sites/kit.svelte.dev/src/lib/search/SearchBox.svelte:234]: A before/after screenshot would be helpful — it's unclear to me what problem this is solving

Comment (Rich-Harris) [inline sites/kit.svelte.dev/src/lib/search/SearchBox.svelte:234]:
```suggestion
		appearance: none;
		-webkit-appearance: none;
```

**Comment by Rich-Harris** [comment]
@bluwy I added `-webkit-appearance` — does it look okay for you now? https://kit-svelte-dev-git-fork-caesar-exploit-patch-1-svelte.vercel.app/

**Comment by bluwy** [comment]
It's still the same unfortunately :( Not sure what's up with Safari

image

I could take a quick look tonight and find a fix.

**Comment by bluwy** [comment]
I think I found the fix. The `appearance: none` (and `-webkit-`) was applied on the wrong search input. It initially was applied on the search box modal.

**Review by bluwy — APPROVED** [review]

**Comment by caesar-exploit** [comment]
Fix, thx guys!

**Pull request merged**